internal/controllers/clusterextension_controller_test.go

```diff
@@ -120,19 +119,17 @@ func TestClusterExtensionNonExistentVersion(t *testing.T) {
 	require.NotNil(t, cond)
 	require.Equal(t, metav1.ConditionFalse, cond.Status)
 	require.Equal(t, ocv1alpha1.ReasonResolutionFailed, cond.Reason)
-	require.Equal(t, fmt.Sprintf(`no package %q matching version "0.50.0" found`, pkgName), cond.Message)
+	require.Equal(t, packageerrors.GenerateVersionError(pkgName, "0.50.0").Error(), cond.Message)
 	cond = apimeta.FindStatusCondition(clusterExtension.Status.Conditions, ocv1alpha1.TypeInstalled)
 	require.NotNil(t, cond)
-	require.Equal(t, metav1.ConditionUnknown, cond.Status)
-	require.Equal(t, ocv1alpha1.ReasonInstallationStatusUnknown, cond.Reason)
-	require.Equal(t, "installation has not been attempted as resolution failed", cond.Message)
+	require.Equal(t, metav1.ConditionFalse, cond.Status)
 
 	verifyInvariants(ctx, t, reconciler.Client, clusterExtension)
 	require.NoError(t, cl.DeleteAllOf(ctx, &ocv1alpha1.ClusterExtension{}))
-	require.NoError(t, cl.DeleteAllOf(ctx, &rukpakv1alpha2.BundleDeployment{}))
 }
 
 func TestClusterExtensionBundleDeploymentDoesNotExist(t *testing.T) {
+	t.Skip("Skip and replace with e2e for BundleDeployment")
```

Comment:
The BD API would be gone, but it was intentional to keep this for now rather than removing it because we want to test a similar scenario with ClusterExtension. This test may not even be relevant and just go away, but keeping the tests as is for now rather than removing them which we will handle in follow up.